File: backend/src/main/java/edu/cit/amihan/medibook/patient/dto/PatientRequest.java
package edu.cit.amihan.medibook.patient.dto;

import jakarta.validation.constraints.NotBlank;
import lombok.Data;

import java.time.LocalDate;

@Data
public class PatientRequest {

    @NotBlank(message = "Full name is required")
    private String fullName;

    private LocalDate dateOfBirth;

    private String contactNumber;

    private String address;
}

File: mobile/app/src/main/java/edu/cit/amihan/medibook/feature/patient/model/PatientRequest.kt
package edu.cit.amihan.medibook.feature.patient.model

import com.google.gson.annotations.SerializedName

data class PatientRequest(
    @SerializedName("fullName") val fullName: String,
    @SerializedName("dateOfBirth") val dateOfBirth: String? = null,
    @SerializedName("contactNumber") val contactNumber: String? = null,
    @SerializedName("address") val address: String? = null
)
